javascriptv3/example_code/personalize/README.md

```diff
@@ -0,0 +1,81 @@
+- [Create an event tracker](src/personalize_createEventTracker.js) (CreateEventTrackerCommand)
+- [Create a filter](src/personalize_createFilter.js) (CreateFilterCommand)
+- [Create a recommender](src/personalize_createRecommender.js) (CreateRecommenderCommand)
+- [Create a schema](src/personalize_createSchema.js) (CreateSchemaCommand)
+- [Create a solution](src/personalize_createSolution.js) (CreateSolutionCommand)
+- [Create a solution version](src/personalize_createSolutionVersion.js) (CreateSolutionVersion)
+- [Get a personalized ranking](src/personalize_getPersonalizedRanking.js) (GetPersonalizedRankingCommand)
+- [Get recommendations (custom dataset group)](src/personalize_getRecommendations.js) (GetRecommendationsCommand)
+- [Get recommendations from a recommender (domain dataset group)](src/personalize_getRecommendationsFromRecommender.js) (GetRecommendationsCommand)
+- [Get recommendation with a filter (custom dataset group)](src/personalize_getRecommendationsWithFilter.js) (GetRecommendationsCommand)
+- [Put events](src/personalize_putEvents.js) (PutEventsCommand)
+- [Put items](src/personalize_putItems.js) (PutItemsCommand)
+- [Put users](src/personalize_putUsers.js) (PutUsersCommand)
+
+**Note**: All code examples are written in ECMAscript 6 (ES6). For guidelines on converting to CommonJS, see 
+[JavaScript ES6/CommonJS syntax](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/sdk-examples-javascript-syntax.html).
+
+## Important
+
+- As an AWS best practice, grant this code least privilege, or only the
+  permissions required to perform a task. For more information, see
+  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege)
+  in the *AWS Identity and Access Management User Guide*.
+- This code has not been tested in all AWS Regions. Some AWS services are
+  available only in specific AWS Regions. For more information, see the
+  [AWS Regional Services List](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
+  on the AWS website.
+- Running this code might result in charges to your AWS account.
+
+## Running the code
+
+### Prerequisites
+- An AWS account. To create an account, see [How do I create and activate a new AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) on the AWS Premium Support website.
+- AWS credentials. For details, see  [Setting credentials in Node.js](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/setting-credentials-node.html) in the
+  *AWS SDK for Javascript (v3) Developer Guide*.
+
+1. Clone the [AWS SDK Code Samples repo](https://github.com/awsdocs/aws-doc-sdk-examples) to your local environment. See [the Github documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) for instructions.
```

Comment:
Update last sentence: 

For instructions, see [the GitHub documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).